embulk-output-redshift/src/main/java/org/embulk/output/RedshiftOutputPlugin.java

```diff
@@ -83,6 +83,14 @@
         @Config("ssl")
         @ConfigDefault("\"disable\"")
         public Ssl getSsl();
+
+        @Config("copy_unload_iam_role_name")
+        @ConfigDefault("\"\"")
+        public String getCopyUnloadIamRoleName();
+
+        @Config("copy_unload_iam_account_id")
```

Comment:
I think '...iam_account...' is a bit confusing. How about 'copy_aws_account_id' for the option name?

Reply:
Exactly, account id and iam has not relation, I fix it just `copy_aws_account_id`.

Reply:
I think it is better to fix name of variables in class `RedshiftCopyBatchInsert` to same as option name, so change them together.